src/index.ts

```diff
@@ -15,11 +15,9 @@ import {Job, cleanupJobResources} from "./job.js";
 
 const jobs: Job[] = [];
 
-process.on("exit", (code) => {
```

Comment:
Seemed like the cleanupJobResources() here does nothing, since for event `exit`, [Listener functions must only perform synchronous operations. ](https://nodejs.org/api/process.html#event-exit)

Reply:
I think you are right.

But this won't fix that the sleep 10 container isn't cleaned up will it ?

Reply:
Nvm, I didn't read the entire change log :smile: